### ISSUE TITLE
.bldr.toml should work with new plan connection

### ISSUE THREAD

**Issue by smacfarlane** [habitat-sh/builder#1202]
Once the work related to #1197 lands, we'll have the ability to connect plans for individual targets. This would give us two ways to control what packages get built and two places that need configuring, the UI (for the front end) and the .bldr.toml (for merge notifications from github).  This is potentially confusing for a user if they've configured the two differently. 

Since we'll be able to control plan connection per-target,  I think it would be useful for the .bldr.toml to be able to auto-configure plan connections for the front end. 

This may require additional work before it is viable, as repositories and plan-connections are not a 1-1 relationship. We may need an additional 'connect repo'-like workflow in order for this behavior to work.
